hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -268,6 +268,12 @@ private FlinkOptions() {
       .withDescription("Payload class used. Override this, if you like to roll your own merge logic, when upserting/inserting.\n"
           + "This will render any value set for the option in-effective");
 
+  public static final ConfigOption<Boolean> PARTIAL_OVERWRITE_ENABLED = ConfigOptions
```

Comment:
What's the idea for this additional configuration (beside the record payload class)?

Reply:
This is the feature toggle to control another change about `BucketAssignFunction` to support the case the record partition path is changed, but I have removed it. so this feature toggle can be removed as well.